config/config-inyo/filter.format.body.yml

```diff
@@ -21,7 +21,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <b> <i class=""> <cite> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class> <h3 id> <h4 id> <h5 id> <h6 id> <table class=""> <img src alt data-entity-type data-entity-uuid data-align data-caption> <drupal-media data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class="jcc-button--primary jcc-button--cta-link"> <blockquote cite class="blockquote blockquote--alert blockquote--alert--warning blockquote--alert--error blockquote--alert--success blockquote--alert--info blockquote--callout blockquote--italic blockquote--box"> <p> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <hr> <video> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button alt title> <iframe longdesc name scrolling src title align height frameborder width>'
+      allowed_html: '<em> <strong> <b> <i class=""> <cite> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class> <h3 id> <h4 id> <h5 id> <h6 id> <table class=""> <img src alt data-entity-type data-entity-uuid data-align data-caption> <drupal-media data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class="jcc-button--primary jcc-button--cta-link radiomast__status--*"> <blockquote cite class="blockquote blockquote--alert blockquote--alert--warning blockquote--alert--error blockquote--alert--success blockquote--alert--info blockquote--callout blockquote--italic blockquote--box"> <p> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <hr> <video> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button alt title> <iframe longdesc name scrolling src title align height frameborder width>'
```

Comment:
`radiomast__status--*`: Could we switch the name class to use dashes instead of underscores? Just to follow convention. :stuck_out_tongue: